cdap-docs/_common/tabbed-parsed-literal.py

```diff
@@ -376,23 +393,30 @@ def cleanup_content(self):
     
         return line_counts, lines
     
-    def cleanup_option(self, option, default):
+    def cleanup_option(self, option, default, ascii_only=False):
         """Removes leading or trailing quotes or double-quotes from a string option."""
         _option = self.options.get(option,'')
         if not _option:
             return default
         else:
-            return dequote(_option)
+            return clean_ascii(dequote(_option)) if ascii_only else dequote(_option)
```

Comment:
Umm. No. We want to first de-quote the option, and then replace any remaining non-alphanumeric characters with hyphens. If you do as you suggest, you'll end up with this:
```
-I-was-enclosed-in-quotes-
```
instead of 
```
I-was-enclosed-in-quotes
```